website/docs/Pipelines/Whole_Genome_Germline_Single_Sample_Pipeline/README.md

```diff
@@ -44,57 +89,235 @@ The [Whole Genome Germline Single Sample workflow](https://github.com/broadinsti
+| preemptible_tries |  PapiSettings (papi_settings) | Number of times the workflow can be preempted. | Int | 
+| agg_preemptible_tries |  PapiSettings (papi_settings) | Number of preemtible machine tries for the BamtoCram task. | Int |
+ 
+#### Additional inputs
+Additional inputs that are not contained in a struct are described in the table below. Similar to the struct inputs, these inputs are specified in the [example configuration files](https://github.com/broadinstitute/warp/tree/develop/pipelines/broad/dna_seq/germline/single_sample/wgs/input_files). 
 
-Learn more about the software tools implemented in these tasks by reading the GATK [data pre-processing](https://gatk.broadinstitute.org/hc/en-us/articles/360035535912) and [germline short variant discovery](https://gatk.broadinstitute.org/hc/en-us/articles/360035535932) documentation.
 
+| Input name | Input description | Input type |
+| --- | --- | --- |
+| fingerprint_genotypes_file | Genotype VCF if optionally performing fingerprinting.  | File |
+| fingerprint_genotypes_index | Optional index for the fingerprinting VCF. | File |
+| wgs_coverage_interval_list | Interval list for the CollectWgsMetrics tool. | File |
+| provide_bam_output | If set to true, provides the aligned BAM and index as workflow output; default set to false. | Boolean |
+| use_gatk3_haplotype_caller | Uses the GATK3.5 HaplotypeCalller; default set to true. | Boolean |
+| dragen_functional_equivalence_mode | Boolean used to run the WGS pipeline in a mode functionally equivalent to DRAGEN; set to false by default. | Boolean |
+| dragen_maximum_quality_mode | Boolean used to run the pipeline in DRAGEN mode with modifications to produce maximum quality results; set to false by default. | Boolean |
+| run_dragen_mode_variant_calling | Boolean used to indicate that DRAGEN mode should be used for variant calling; default set to false but must be true to compose DRAGstr model and perform variant calling with HaplotypeCaller in dragen-mode. | Boolean |
+| use_spanning_event_genotyping | Boolean used to call the HaplotypeCaller --disable-spanning-event-genotyping parameter; default set to true so that variant calling includes spanning events. Set to false to run the DRAGEN pipeline.  | Boolean |
+| unmap_contaminant_reads | Boolean to indicate whether to identify extremely short alignments (with clipping on both sides) as cross-species contamination and unmap the reads; default set to true. This feature is not used in the pipeline mode functionally equivalent to DRAGEN. | Boolean |
+| perform_bqsr | Boolean to turn on base recalibration with BQSR; default set to true, but not necessary when running the pipeline in DRAGEN mode. | Boolean |
+| use_bwa_mem | Boolean indicating if workflow should use the BWA mem aligner; default set to true, but must be set to false to alternatively run the DRAGEN-GATK DRAGMAP aligner. | Boolean | 
```

Comment:
Do we have documentation that lists all the valid combinations of inputs and why you might want to configure it different ways? 

It's late in the life of this PR to bring this up, but if we are validating and supporting only 3 modes of the pipeline for instance, would it make sense to abstract these inputs into a single input, for instance `pipeline_mode` that takes 1 of 3 options (for instance ` GATK_best_prcatices`, `DRAGEN_functional_equivalence`, or `DRAGEN_maximu_quality`) and then these variables are all set by the workflow to the appropriate values for that mode?